modules/gapi/include/opencv2/gapi/infer/ov.hpp

```diff
@@ -0,0 +1,884 @@
+        auto &model = cv::util::get<detail::ParamDesc::Model>(m_desc.kind);
+        model.mean_values = std::move(mean_vec);
+        return *this;
+    }
+
+    /** @overload
+
+    @param mean_map Map of pairs: name of corresponding input layer
+    and its mean values.
+    @return reference to this parameter structure.
+    */
+    Params<Net>& cfgMean(detail::AttrMap<std::vector<float>> mean_map) {
+        if (cv::util::holds_alternative<detail::ParamDesc::CompiledModel>(m_desc.kind)) {
+            cv::util::throw_error(
+                    std::logic_error("Specifying mean values isn't"
+                                     " possible for compiled model."));
+        }
+        GAPI_Assert(cv::util::holds_alternative<detail::ParamDesc::Model>(m_desc.kind));
+        auto &model = cv::util::get<detail::ParamDesc::Model>(m_desc.kind);
+        model.mean_values = std::move(mean_map);
+        return *this;
+    }
+
+    /** @brief Specifies scale values for preprocessing.
+     *
+    The function is used to set scale values for input layer preprocessing.
+
+    @param scale_values Float vector contains scale values
+    @return reference to this parameter structure.
+    */
+    Params<Net>& cfgScale(std::vector<float> scale_values) {
+        if (cv::util::holds_alternative<detail::ParamDesc::CompiledModel>(m_desc.kind)) {
+            cv::util::throw_error(
+                    std::logic_error("Specifying scale values isn't"
+                                     " possible for compiled model."));
+        }
+        GAPI_Assert(cv::util::holds_alternative<detail::ParamDesc::Model>(m_desc.kind));
+        auto &model = cv::util::get<detail::ParamDesc::Model>(m_desc.kind);
+        model.scale_values = std::move(scale_values);
+        return *this;
+    }
+
+    /** @overload
+
+    @param scale_map Map of pairs: name of corresponding input layer
+    and its mean values.
+    @return reference to this parameter structure.
+    */
+    Params<Net>& cfgScale(detail::AttrMap<std::vector<float>> scale_map) {
+        if (cv::util::holds_alternative<detail::ParamDesc::CompiledModel>(m_desc.kind)) {
+            cv::util::throw_error(
+                    std::logic_error("Specifying scale values isn't"
+                                     " possible for compiled model."));
+        }
+        GAPI_Assert(cv::util::holds_alternative<detail::ParamDesc::Model>(m_desc.kind));
+        auto &model = cv::util::get<detail::ParamDesc::Model>(m_desc.kind);
+        model.scale_values = std::move(scale_map);
+        return *this;
+    }
+
+    /** @brief Specifies resize interpolation algorithm.
+     *
+    The function is used to configure resize preprocessing for input layer.
+
+    @param interpolation Resize interpolation algorithm.
+    Supported algorithms: #INTER_NEAREST, #INTER_LINEAR, #INTER_CUBIC.
+    @return reference to this parameter structure.
+    */
+    Params<Net>& cfgResize(int interpolation) {
```

Comment:
Should we add a default value here?

Reply:
Well, probably but then there won't be any default value for `map` case?